style.css

```diff
@@ -1,250 +1,575 @@
-background: url("./images/fon.png");
-margin-top: -131px;
-margin-left: -10px;
+ position: inherit;
+ margin-bottom: 55px;
 }
 
-.middle__title {
-color: red;
-font-weight: 600;
-font-size: 90px;
-line-height: 10px;
-margin-left: 100px;
-margin-top:100px;
+.top__avto{
+  max-width: 100%;
 }
+
 .other {
-display: flex;
-justify-content: space-around;
-flex-flow: row nowrap;
-width: 1000px;
-height: 40px;
-margin-left: 200px;
+  display: flex;
+  justify-content: space-around;
+  height: 40px;
+  padding-right: 356px;
+  padding-left: 309px;
+  padding-bottom: 30px;
+}
+
+.other__block1 {
+  display: inline-block;
+
+}
+
+.other__speedometr {
+ position: inherit;
+}
+
+.other__kontur {
+  display: block;
+  margin-right: 10px;
+}
+
+.other__tab {
+  position: relative;
+  left: 16px;
+  bottom: 31px;
+}
+
+.other__strel {
+  position: relative;
+  bottom: 34px;
+  right: 2px;
+}
+
+.other__text-left {
+  display: flex;
+  flex-direction: column;
+}
+
+.other__left {
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.other__sec {
+  position: relative;
+  left: 55px;
+  bottom: 23px;
+}
+
+.other__bottom-l {
+  position: relative;
+  font-size: 12px;
+  line-height: 15px;
+  bottom: 17px;
+}
+
+.other__center {
+  position: inherit;
+}
+
+.other__vector5 {
+  position: absolute;
+  z-index: 2;
+}
+
+.other__trassa {
+  position: relative;
+  right: 2px;
+  z-index: 1;
+}
+
+.other__ellipse-top {
+  position: relative;
+  bottom: 42px;
+  right: 13px;
+  z-index: 2
+}
+
+.other__ellipse-bot {
+  position: relative;
+  right: 43px;
+  top: 1px;
+  z-index: 2
+}
+
+.other__point-top {
+  position: relative;
+  right: 36px;
+  bottom: 44px;
+  z-index: 3
+}
+
+.other__point-bot {
+  position: relative;
+  right: 62px;
+  bottom: 1px;
+  z-index: 3
+}
+
+.other__next-center {
+  display: flex;
+  flex-direction: column;
+}
+
+.other__text-center {
+  font-size: 35px;
+  line-height: 43px;
+}
+
+.other__km {
+  position: relative;
+  left: 68px;
+  bottom: 23px;
+}
+
+.other__bottom-с {
+  position: relative;
+  font-size: 12px;
+  line-height: 15px;
+  bottom: 17px;
+}
+
+.other__icon {
+  display: inline-flex;
+  align-items: center;
+}
+
+.other__link_testdrive {
+  position: relative;
+  text-decoration: none;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  color: #000000;
+}
+
+.other__vector3 {
+  position: relative;
+  bottom: 5px;
+}
+
+.other__vector4 {
+  position: relative;
+  right: 19px;
+  bottom: 1px;
+}
+.other__point3 {
+  position: relative;
+  right: 29px;
+  bottom: 7px;
 }
 
 .other__click {
-display: flex;
-align-items: center;
-width: 160px;
-height: 40px;
-border: 2px solid #E51937;
-box-sizing: border-box;
-color: #E51937;
-background: #ffffff;
-margin-left: -100px;
+  font-weight: bold;
+  font-size: 13px;
+  line-height: 16px;
+  text-align: center;
+  width: 160px;
+  height: 40px;
+  border: 2px solid #e51937;
+  box-sizing: border-box;
+  color: #e51937;
+  background: #fff;
+  margin-bottom: 15px; 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)